### PR TITLE
NXTGEN-584 Now properly indexing properties according to 'https://luc…

### PR DESCRIPTION
**Pull request by EthanEiter**

### CODE REVIEW

File: src/NHibernate.Search/Bridge/String2FieldBridgeAdaptor.cs [hunk 1]
```diff
@@ -18,15 +18,24 @@ public String2FieldBridgeAdaptor(IStringBridge stringBridge)
...
             if (StringHelper.IsNotEmpty(indexedString))
             {
-                var field = new StringField(name, indexedString, store);
+                var storeType = store == Field.Store.YES ? TextField.TYPE_STORED : TextField.TYPE_NOT_STORED;
+
+                var fieldType = new FieldType(storeType);
+
+                if (omitNorms != null)
+                {
+                    fieldType.OmitNorms = omitNorms.Value;
```
Comment (codekaizen) [inline src/NHibernate.Search/Bridge/String2FieldBridgeAdaptor.cs:35]: ```fieldType.OmitNorms = omitNorms ?? false;```

Comment (EthanEiter) [reply src/NHibernate.Search/Bridge/String2FieldBridgeAdaptor.cs:35]: According to the upgrade doc I was following, having it not set the OmitNorms is a thing:

> If you did this before (value can be String or Reader):

>new Field("field", value, Field.Store.NO, Field.Indexed.ANALYZED)
>you can now do this:

>new TextField("field", value)

Comment (EthanEiter) [reply src/NHibernate.Search/Bridge/String2FieldBridgeAdaptor.cs:35]: As opposed to actually setting the field:

> new Field("field", "value", Field.Store.YES, Field.Indexed.NOT_ANALYZED)
> you can now do this:

> FieldType ft = new FieldType(StringField.TYPE_STORED);
> ft.setOmitNorms(false);
> new Field("field", "value", ft)

**Comment by joshluth** [comment]
Super old.. closing this down

**Pull request closed without merging**